src/Core/Framework/DataAbstractionLayer/Write/WriteCommandExtractor.php

```diff
@@ -337,7 +338,11 @@ private function getAssociationByStorageName(string $name, array $fields): ?Many
      */
     private function getMainFields(array $fields): array
     {
-        $main = [];
+        $main = [
```

Comment:
i would rather add an instance check for this two fields to the loop and them then to the main array, like it's the case with the FKField.

Because some definitions don't have an updatedAt field (MappingDefinitions)

Reply:
Okay, I have changed this.

Reply:
@keulinho push